agents/ibm_powervs/fence_ibm_powervs.py

```diff
@@ -265,18 +289,18 @@ def main():
 	define_new_opts()
 
 	all_opt["shell_timeout"]["default"] = "500"
-	all_opt["power_timeout"]["default"] = "30"
-	all_opt["power_wait"]["default"] = "1"
-	all_opt["stonith_status_sleep"]["default"] = "2"
+	all_opt["power_timeout"]["default"] = "120"
+	all_opt["power_wait"]["default"] = "15"
+	all_opt["stonith_status_sleep"]["default"] = "10"
 	all_opt["api-type"]["default"] = "private"
 	all_opt["proxy"]["default"] = ""
 
 	options = check_input(device_opt, process_input(device_opt))
 
 	docs = {}
 	docs["shortdesc"] = "Fence agent for IBM PowerVS"
-	docs["longdesc"] = """fence_ibm_powervs is a Power Fencing agent which can be \
-used with IBM PowerVS to fence virtual machines."""
+	docs["longdesc"] = """fence_ibm_powervs is a fencing agent for \
```

Comment:
It should still say Power Fencing agent here (Power fencing is when a node is power-cycled, reset, or turned off to prevent it from issuing I/Os).

For full list of different fencing definitions see https://github.com/ClusterLabs/fence-agents/blob/main/doc/FenceAgentAPI.md.

Remember to run `make xml-upload` to also update the test-metadata after changing it.

Reply:
done